service.upnext/addon.xml

```diff
@@ -5,15 +5,14 @@
     </requires>
     <extension library="service.py" point="xbmc.service"/>
     <extension point="xbmc.addon.metadata">
-        <summary lang="en_GB">Provides a netflix style pop up notification for watching the next episode with fancy/simple
```

Comment:
I didn't tell to remove the lang attributes.

Reply:
Ahh ok I'll put them back

Reply:
I asked for the removal of the language element from metadata not the attributes for summary and description